addons/stock_account/migrations/13.0.1.1/post-migration.py

```diff
@@ -0,0 +1,224 @@
+# Copyright 2020 ForgeFlow <http://www.forgeflow.com>
+# Copyright 2020 Andrii Skrypka
+# Copyright 2021 Tecnativa - Carlos Dauden
+# Copyright 2021 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from openupgradelib import openupgrade
+from odoo import _
+from odoo.tools.float_utils import float_compare, float_is_zero, float_round
+from odoo.addons.base.models.ir_model import query_insert
+
+_logger = logging.getLogger(__name__)
+
+# Declare global variant to avoid that it is passed between methods
+precision_price = 0
+
+
+def _prepare_common_svl_vals(move, product):
+    return {
+        "create_uid": move["write_uid"],
+        "create_date": move["date"],
+        "write_uid": move["write_uid"],
+        "write_date": move["date"],
+        "stock_move_id": move["id"],
+        "company_id": move["company_id"],
+        "product_id": move["product_id"],
+        "description": move["reference"] and "%s - %s" % (move["reference"], product.name) or product.name,
+        "unit_cost": 0.0,
+        "remaining_qty": 0.0,
+        "remaining_value": 0.0,
```

Comment:
```suggestion
        "remaining_value": 0.0,
        "old_product_price_history_id": price_history_rec["id"],
```

Reply:
but _prepare_common_svl_vals doesn't have price_history_rec

Reply:
I mean, I am not sure how I should add price_history_rec in the `generate_stock_valuation_layer` method for in / out svl. Should I use a `while` like is done in the other cases or what?

Reply:
@MiquelRForgeFlow I think he mistook and it has to be in `_prepare_man_svl_vals`

Reply:
thanks

Reply:
Yes sorry, it was an error passing the line from my editor to the comment in the PR